<launch>
    <!-- Launch Stage 1 -->
    <node name="stage1_node" pkg="miro_pose_imitation" type="stage1_node.py" output="screen">
        <param name="use_sim_time" value="false" />
    </node>

    <!-- Launch Main Controller (waits for stage1 to complete) -->
    <node name="main_controller" pkg="miro_pose_imitation" type="main_controller.py" output="screen">
        <param name="use_sim_time" value="false" />
    </node>
</launch>
